tests/core/vm/test_computation.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+from evm.vm import (
+    Computation,
+    Message,
+)
+
+from eth_utils import (
+    to_canonical_address,
+)
+
+from evm.exceptions import (
+    ValidationError,
+)
+
+CANONICAL_ADDRESS_1 = to_canonical_address("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")
+CANONICAL_ADDRESS_2 = to_canonical_address("0xcd1722f3947def4cf144679da39c4c32bdc35681")
+
+
+@pytest.fixture
+def message():
+    message = Message(
+        origin=to_canonical_address(CANONICAL_ADDRESS_2),
+        to=CANONICAL_ADDRESS_1,
+        sender=to_canonical_address(CANONICAL_ADDRESS_2),
+        value=100,
+        data=b"0x",
+        code=b"0x",
+        gas=100,
+        gas_price=1,
+    )
+    return message
+
+
+@pytest.fixture
+def computation(message):
+    computation = Computation(
+        vm=1,
```

Comment:
Just a note-to-self:

Giving the computation full access to the VM is probably an antipattern as computation should really only require very finite access to a finite number of APIs.  It would be very nice if we could get the `vm` object out of the computation in a clean way.